src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -97,6 +96,12 @@ public void setupRoutes(Router router) {
             }
         }, Role.ADMINISTRATOR));
 
+        router.post("/api/operator/updatePublicPrivateOperator").blockingHandler(auth.handle((ctx) -> {
```

Comment:
URLs should not use capitals - different web servers handle casing differently so it tends to create confusion. The convention tends to be to use dashes instead.

Reply:
noted have removed this and just re-use the good old /update end point altogether.